### PR TITLE
DI-174 Investigate and agree reporting approach

### PR DESCRIPTION
**Pull request by JackPlowman**
## Link to JIRA Ticket

- https://nhsd-jira.digital.nhs.uk/browse/DI-174

## Description

This is the ADR for the reporting spike DI-174

## Type of change

- This change requires a documentation update

### CODE REVIEW

**Review by rohith1122 — APPROVED** [review]
LGTM

**Review by mik3ola — DISMISSED** [review]

**Review by rohith1122 — APPROVED** [review]
LGTM

File: documentation/adr/002-Reporting-approach.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+* Difficult to get access to Splunk. Slow and very restrictive permissions.
+* Splunk’s limited job search rate
+* To generate a report the user must understand how to write Splunk queries
+* Control of our reporting is in Splunk's hands
+* Logs may not reach Splunk (very low possibility)
+* Splunk holds log data for only two years (another storage method is needed for longer than two years)
+
+### Option 2 - Use AWS Services for Reporting, Emailing and Alerting
+
+This option is to use a range of AWS services to store logs, report, email and alert. Such as S3, Athena, EventBridge, Lambda and SES. This isn't a definite list but some possible services that can be used to create the solution.
```
Comment (nat-darke-nhs) [inline documentation/adr/002-Reporting-approach.md:49]: change to "definitive list"

**Review by nat-darke-nhs — APPROVED** [review]
I made a couple of comments but approved. Good stuff. Well thought through and the right decision, in my opinion.

**Review by rohith1122 — DISMISSED** [review]
LGTM

File: documentation/adr/002-Reporting-approach.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+### Detailed analysis of the options
+
+#### Option 1 - Splunk Reporting, Emailing and Alerting
+
+This option uses Cloud Splunk as a platform for reporting. Using all their features such as generating reports, emailing reports and alerting from Splunk.
+
+Pros:
+
+* Splunk is a known product so won't require validation
```
[the remark this answers is not in the document]
Comment (JackPlowman) [reply documentation/adr/002-Reporting-approach.md]: Updated, please can you take a look again and if it now makes more sense.

**Review by nat-darke-nhs — APPROVED** [review]

**Review by rohith1122 — APPROVED** [review]

**Pull request merged**